client/src/components/Verify.js

```diff
@@ -118,8 +121,9 @@ class Verify extends Component {
    */
   handleCompleteVerification(verifyData) {
     const { endpoints: { verify }, onCompleteVerification } = this.props;
-    const { selectedMethod } = this.state;
-    const endpoint = verify.replace('{urlSegment}', selectedMethod.urlSegment);
+    const { selectedMethod, token } = this.state;
+    const params = token ? `?SecurityID=${token}` : '';
```

Comment:
You can provide the SecurityID in the post body, doesn't have to be in the query string

Reply:
I _could_ do this. I'll have to (JSON) decode the request on the server-side. Currently the method implementations get to define what the request payload is (in their React components) and then get to decode the request payload themselves (in their `VerifyHandler` implementation).

I decided I wouldn't bother changing that and all the methods to match.

Reply:
Good justification